pkg/service/service.go

```diff
@@ -57,7 +57,7 @@ func Remove(ctx context.Context, client *containerd.Client, name string) error {
 
 	} else {
 		service := client.SnapshotService("")
-		key := name + "snapshot"
+		key := name + "-snapshot"
```

Comment:
Can you link to where this key is used in the codebase?

Please see the contributing guide. Where is the issue that shows how to reproduce and observe this problem?

Reply:
Ok, you can look [here](https://github.com/openfaas/faasd/blob/70b0929cf25d61bcb9855118332cc6a6e6628c23/pkg/supervisor.go#L220)  and [here](https://github.com/openfaas/faasd/blob/70b0929cf25d61bcb9855118332cc6a6e6628c23/pkg/provider/handlers/deploy.go#L164).
In some cases, deleting a snapshot with the wrong key will fail, resulting in the failure to create the faas container the next time.